week1/index.html

```diff
@@ -0,0 +1,129 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="favicon.svg" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css">
+    <title>help hamsick</title>
+  </head>
+
+  <body>
+    <header>
+      <div class="headerColorBox">
+      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
+      </div>
+    </header>
```

Comment:
```suggestion
    <header class="header">
      <h1>💛🍔 햄식이를 도와줘 🍔💛</h1>
    </header>
```
이런 식으로 header 테그 자체를 wrapper로 이용해도 좋을 것 같아요! semantic tag가 wrapper역할을 할 수 있다면, 불필요한 div테그를 사용하지 않아도 좋을 것 같습니다 : )

Reply:
오 header를 wrapper로 사용할 수 있는 지는 몰랐어요! 지식 +1해갑니당